### PR TITLE
Basic commands

### LINKED ISSUE #9: Basic sql commands
- [x] Select command.
- [x] Insert command.

### PR DESCRIPTION
**Pull request by shiryz**
Fixes #9

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,104 @@
...
+
+  Ownership table:
+
+  |id|owner_id|pet_id|
+  |--|--------|------|
+
 ## Basic commands for selecting and inserting.
+
+- Assuming you have a database ready with some data in it, how would you get the data from the database, explain the structure of a select query:
+Select [columns (separated by a comma)] From [table]
```
Comment (ZooeyMiller) [inline README.md]: I would add the necessity of a semicolon to finish a statement, just for clarity's sake.

**Review by ZooeyMiller — CHANGES_REQUESTED** [review]
as above :arrow_up:

File: README.md (hunk 1 above)
Comment (shiryz) [reply README.md]: good point :+1:

**Review by ZooeyMiller — APPROVED** [review]
:tada: awesome

**Pull request closed without merging**